File: Module01_PythonBasics/problem01.py
def csWhereIsBob(names):
     '''
     input names - a list of strings
                        
     output is an integer which is the location of Bob 
     if Bob not present return -1
     '''
                                                
     if "Bob" not in names:
         return -1
                                                                    
     for i in range(len(names)):
         if names[i] == "Bob":
             return i

print(csWhereIsBob(["Jimmy", "Layla", "Bob"])) # 2
print(csWhereIsBob(["Bob", "Layla", "Kaitlyn", "Patricia"])) # 0
print(csWhereIsBob(["Jimmy", "Layla", "James"])) # -1

File: Module01_JavaBasics/Problem01.java
/* input names - a list of strings
 *
 * output - an integer which is the location of Bob
 * if Bob not present return -1
 */ 

import java.util.ArrayList;


public class Problem01 {

    public static void main(String[] args) {
	
	String[] names_list = new String[] {"Jeff", "Kim", "Bob"};
	
	String name = "Bob";

	for (int i=0; i<names_list.length; i++) {
	    if (names_list[i] == name) {
	        System.out.println(name);
		System.out.println(i);
	    }
	    else {
	    	System.out.println("Bob is not here?");
	    }
	}
    }

}
